python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -402,6 +419,14 @@ def selectPfsConfig(pfsConfig: PfsConfig, flagName: str, isGood: Sequence[bool])
             return goodConfig
 
         pfsConfig = selectPfsConfig(pfsConfig, "ABSENT_FIBER", np.isin(pfsConfig.fiberId, pfsMerged.fiberId))
+
+        pfsMerged = removeBadSpectra(
+            pfsMerged,
+            self.config.cutoffSNR,
+            (self.config.cutoffSNRRangeLeft, self.config.cutoffSNRRangeRight),
+        )
```

Comment:
Need to check the case that `len(pfsMerged) = 0` after calling `removeBadSpectra`.

Reply:
I don't think so. That condition is checked tens of lines below, before pfsMerged is actually used.